src/api/app/components/bs_request_comment_component.rb

```diff
@@ -3,13 +3,19 @@
 # It is used in the beta view of the request show page, under the Overview tab,
 # merged with the BsRequestHistoryElementComponent.
 class BsRequestCommentComponent < ApplicationComponent
-  attr_reader :comment, :commentable, :level
+  attr_reader :comment, :commentable, :level, :diff
 
-  def initialize(comment:, commentable:, level:)
+  def initialize(comment:, commentable:, level:, diff: nil)
     super
 
     @comment = comment
     @commentable = commentable
     @level = level
+    @diff = diff
+  end
+
+  def range
+    line_index = @comment.diff_ref.match(/diff_([0-9]*)_n([0-9]*)/).captures.second
```

Comment:
```suggestion
    line_index = @comment.diff_ref.match(/diff_[0-9]+_n([0-9]+)/).captures.first
```
- We can replace here the `*`s by the `*`s. We don't need to take in account the case of not having a digit.
- We don't need to extract the first digits in the string, so we can remove the parenthesis and change `.first` by `.second`.

What do you think?